app/routes/app/brainstorm/s/$id.tsx

```diff
@@ -192,7 +193,7 @@ export default function ChallengeSubmission() {
           <div className="flex flex-col-reverse md:flex-row space-y-reverse space-y-7 gap-x-5">
             <main className="flex-1">
               <div className="w-full border-spacing-4 border-separate space-y-5">
-                {reviews.map((m) => {
+                {reviews.map((m: Review) => {
```

Comment:
The type for `reviews` be inferred automatically, so i don't think the manual typing is necessary.